src/tabs/tabs.component.ts

```diff
@@ -89,6 +91,14 @@ export class Tabs implements AfterContentInit {
 	 * Set to `true` to have the tabIndex of the all tabpanels be -1.
 	 */
 	@Input() isNavigation = false;
+	/**
+	 * Sets the aria label on the `TableHeader`'s nav element.
```

Comment:
`TabHeader`s ... no `'` needed either. Also on the comment for `ariaLabelledby`

```suggestion
	 * Sets the aria label on the `TabHeader`s nav element.
```